Easydict/NewApp/View/SettingView/SettingView.swift

```diff
@@ -51,15 +51,26 @@ struct SettingView: View {
 
     func resizeWindowFrame() {
         guard let window else { return }
+        
+        // Disable zoom button, ref: https://stackoverflow.com/a/66039864/8378840
+        window.standardWindowButton(.zoomButton)?.isEnabled = false
```

Comment:
It seems that disabling the zoom button in SwiftUI is not easy, I found a workaround from here  https://stackoverflow.com/a/66039864/8378840

Please let me know if there is a better solution.

<img width="686" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/6c8e97a7-008c-4ff6-afc4-a2ae8fc36555">

Reply:
Looks good.